LayoutTests/fast/custom-elements/scoped-custom-element-registry-disabled.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html><!-- webkit-test-runner [ ScopedCustomElementRegistryEnabled=true ] -->
+<html>
+<body>
+<script src="../../resources/js-test.js"></script>
+<script>
+
+description('This tests that attachShadow does not call "registry" attribute getter when scoped custom element registry is disabled');
+
+let registryGetterWasCalled = false;
+document.createElement('div').attachShadow({
+    mode: 'closed',
+    get registry2() {
```

Comment:
Is `registry2` correct here? The problematic property is `registry` not `registry2`